elementary/monitor/alerts/alert_messages/builder.py

```diff
@@ -274,7 +284,9 @@ def _get_result_blocks(
             result_blocks.append(
                 CodeBlock(text=result_message.strip()),
             )
-        if result_sample:
+        if (
```

Comment:
unrelated - made more aligned with the way it was before: anomalous value should be considered as the result sample if present